Irrgarten/src/irrgarten/Game.java

```diff
@@ -0,0 +1,112 @@
+package irrgarten;
+import java.util.ArrayList;
+
+public class Game {
+    public static int MAX_ROUNDS = 10;
+    private int currentPlayerIndex;
+    private String log;
+
+    private Player currentPlayer;
+    private Labyrinth labyrinth;
+    private ArrayList<Monster> monsters;
+    private ArrayList<Player> players;
+
+    // Parametrized
+    public Game(ArrayList<Player> players, ArrayList<Monster> monsters) {
```

Comment:
```suggestion
    public Game (int nplayers){
```
En el diagrama de clases especifica la cabecera del método así, creo que no es necesario pasarle los parámetros de jugadores y monstruos (solo iniciarlos dentro) ya que luego en configureLabyrinth() se ponen